python/pyproject.toml

```diff
@@ -140,6 +145,7 @@ all = ["transformers", "sentence-transformers", "qdrant-client", "chromadb", "py
 
 azure = ["azure-ai-inference", "azure-search-documents", "azure-core", "azure-identity", "azure-cosmos", "msgraph-sdk"]
 chromadb = ["chromadb"]
+google = ["google-generativeai"]
```

Comment:
should this `google` be added to `[tool.poetry.extras]` so that it can be specified via a pip command?

Reply:
I think I did this already.

Reply:
I don't see "google" here:

![image](https://github.com/user-attachments/assets/3257df9f-e10f-4427-a243-d85a38ce5d6d)

Does it need to be?

Reply:
I am not sure if it needs to be in the `all` list. I will add it to there for consistency.

Reply:
doing that gives the option for someone to do `pip install semantic-kernel[google]` if they wanted to.